code/modules/robotics/bot/bot_parent.dm

```diff
@@ -90,3 +90,11 @@
 					return 1
 				else
 					return 0
+
+/obj/machinery/bot/examine()
```

Comment:
The subtypes all implemented this individually as copypaste from eachother. I moved it here and removed it from the subtypes.